### ISSUE TITLE
Please archive project if it has been abandoned. 

### ISSUE THREAD

**Issue by BaileyJM02** [github/personal-website#437]
Hi all, as there have been no commits to this project, and it appears there is no one actively reviewing the repository as there are a number of spam issues, could we archive this project to signify to others that this is no longer looked at by GitHub staff and thus no longer maintained.

**Comment by iamirulofficial** [comment]
True, a lot of spam issues and also no one to check the existing issues and PR's

**Comment by dylmye** [comment]
I'm planning to look through the issues here and integrate them + legit PRs into [better-personal-website/personal-website](https://github.com/better-personal-website/personal-website). Feel free to help out!

**Comment by nazir-hussain** [comment]
ok check it
